code/client/src/sdk/mafia/database/c_ui_database.cpp

```diff
@@ -0,0 +1 @@
+#include "sdk/mafia/database/c_ui_database.h"
```

Comment:
I don't think we were clear here:

If the import is in SDK (`code/client/src/sdk/...)`, you must use relative imports (`.../.../`) elsewhere in the code you use absolute imports (as you did well in word_debug.cpp)

Reply:
Here you can use #include "c_ui_database.h"

Reply:
I much prefer not use use relative imports, in my opinion it polutes the code base and generally causes imports to become unreadable. I'd rather use absolute paths starting with 'sdk'

Reply:
It's fine in CPP files and the accompanying header file such as c_ui_database.h for c_ui_database.cpp though.

Reply:
I agree, but there was a discussion about it: https://github.com/MafiaHub/MafiaMP/pull/56#discussion_r1456180848

@Segfaultd ready to reconsider your point of view on this?

Reply:
If 'sdk' is its own contained module which is expected to transfer between projects, then absolute paths should be okay?

Reply:
We should revise using relative paths and switch to absolute ones; it's still portable as long as you set it to include dirs up in whatever project 'SDK' gets moved into. With relative paths, you introduce some extra friction whenever you need to refactor the source file elsewhere. In general, absolute paths give you a better idea of where the various included files are. In contrast, here it's a sea of `../../../` prefixes that force you to build a mental map of the layout instead.

cc @Segfaultd